ziti/cmd/ziti/constants/constants.go

```diff
@@ -50,53 +50,57 @@ const (
+	ZitiCtrlIdentityCertVarDescription               = "Path to Identity Cert for Ziti Controller"
+	ZitiCtrlIdentityServerCertVarName                = "ZITI_CTRL_IDENTITY_SERVER_CERT"
+	ZitiCtrlIdentityServerCertVarDescription         = "Path to Identity Server Cert for Ziti Controller"
+	ZitiCtrlIdentityKeyVarName                       = "ZITI_CTRL_IDENTITY_KEY"
+	ZitiCtrlIdentityKeyVarDescription                = "Path to Identity Key for Ziti Controller"
+	ZitiCtrlIdentityCAVarName                        = "ZITI_CTRL_IDENTITY_CA"
+	ZitiCtrlIdentityCAVarDescription                 = "Path to Identity CA for Ziti Controller"
+	ZitiSigningCertVarName                           = "ZITI_SIGNING_CERT"
+	ZitiSigningCertVarDescription                    = "Path to the Ziti Signing Cert"
+	ZitiSigningKeyVarName                            = "ZITI_SIGNING_KEY"
+	ZitiSigningKeyVarDescription                     = "Path to the Ziti Signing Key"
+	ZitiRouterIdentityCertVarName                    = "ZITI_ROUTER_IDENTITY_CERT"
+	ZitiRouterIdentityCertVarDescription             = "Path to Identity Cert for Ziti Router"
+	ZitiRouterIdentityServerCertVarName              = "ZITI_ROUTER_IDENTITY_SERVER_CERT"
+	ZitiRouterIdentityServerCertVarDescription       = "Path to Identity Server Cert for Ziti Router"
+	ZitiRouterIdentityKeyVarName                     = "ZITI_ROUTER_IDENTITY_KEY"
+	ZitiRouterIdentityKeyVarDescription              = "Path to Identity Key for Ziti Router"
+	ZitiRouterIdentityCAVarName                      = "ZITI_ROUTER_IDENTITY_CA"
+	ZitiRouterIdentityCAVarDescription               = "Path to Identity CA for Ziti Router"
+	ZitiCtrlListenerAddressVarName                   = "ZITI_CTRL_LISTENER_ADDRESS"
+	ZitiCtrlListenerAddressVarDescription            = "The Ziti Controller Listener Address"
+	ZitiCtrlAdvertisedAddressVarName                 = "ZITI_CTRL_ADVERTISED_ADDRESS"
+	ZitiCtrlAdvertisedAddressVarDescription          = "The Ziti Controller Advertised Address"
+	ZitiEdgeCtrlListenerHostPortVarName              = "ZITI_CTRL_EDGE_LISTENER_HOST_PORT"
+	ZitiEdgeCtrlListenerHostPortVarDescription       = "Host and port of the Ziti Edge Controller Listener"
+	ZitiEdgeCtrlAdvertisedHostPortVarName            = "ZITI_EDGE_CTRL_ADVERTISED_HOST_PORT"
+	ZitiEdgeCtrlAdvertisedHostPortVarDescription     = "Host and port of the Ziti Edge Controller API"
+	ZitiEdgeCtrlAdvertisedPortVarName                = "ZITI_EDGE_CONTROLLER_PORT"
+	ZitiEdgeCtrlAdvertisedPortVarDescription         = "The advertised port of the edge controller"
+	ZitiEdgeIdentityEnrollmentDurationVarName        = "ZITI_EDGE_IDENTITY_ENROLLMENT_DURATION"
+	ZitiEdgeIdentityEnrollmentDurationVarDescription = "The identity enrollment duration in minutes"
+	ZitiEdgeRouterEnrollmentDurationVarName          = "ZITI_EDGE_ROUTER_ENROLLMENT_DURATION"
+	ZitiEdgeRouterEnrollmentDurationVarDescription   = "The router enrollment duration in minutes"
```

Comment:
These variable names seem a bit wordy, any suggestions for a shorter name while still being descriptive?